.travis.yml

```diff
@@ -2,6 +2,8 @@
 # RVM/bundler/ruby and whatnot. Right now 'rust' as a language actually
 # downloads a rust/cargo snapshot, which we don't really want for building rust.
 language: c
+compiler: /usr/bin/gcc-4.7
```

Comment:
> The problem is that compiler gets injected after the environment variables get set:

I'm.. not sure I understand why this is a problem. What's the failure mode if you move `CC` next to `CXX` below?